src/js/player.js

```diff
@@ -2218,27 +2219,23 @@ class Player extends Component {
    *         The current video source when getting
    */
   src(source) {
-    if (source === undefined) {
```

Comment:
Do we want to wrap the `this.error` call in a `setTimeout` like what was done in `sourceList_`?